src/quantum/README.rst

```diff
@@ -0,0 +1,161 @@
+Microsoft Azure CLI 'quantum' Extension
+==========================================
+
+Azure Quantum is the first open Quantum computing platform. It offers a range of services
+from quantum hardware to full-state simulators and quantum inspired optimizations,
+providing developers and customers access to the most competitive quantum offering
+on the market.
+
+To learn more about azure quantum visit:
+https://azure.microsoft.com/en-us/services/quantum/
+
+To learn more about quantum computing and Microsoft's Quantum Development Kit visit:
+https://docs.microsoft.com/quantum/
+
+
+# Using the `az quantum` extension to list and manage jobs in Azure Quantum
+
+1. Log in to Azure using your credentials.
+   ```dotnetcli
+   az login
+   ```
+>[!NOTE] In case you have more than one subscription associated with your Azure account you must specify the 
+>subscription you want to use. You can do this with the command `az account set -s <Your subscription ID>`.
+
+2. You can see all the Azure Quantum workspaces in your subscription with the `az quantum workspace list` command:
+   ```dotnetcli
+   az quantum workspace list
+   ```
+
+3. You can use `quantum workspace set` to select a default workspace you want to use 
+   to list and submit jobs. Note that you also need to specify the resource group.
+   ```dotnetcli
+    az quantum workspace set -g MyResourceGroup -w MyWorkspace  -o table
```

Comment:
I need to know how to create a quantum workspace first.

Reply:
I get this error
```
cli.azure.cli.core.azclierror: ResourceNotFoundError: InvalidResourceType: The resource type could not be found in the namespace 'Microsoft.Quantum' for api version '2019-11-04-preview'.
```
Have you met it before? What should I do to be able to use it? Add my subscription to allowed list?

Reply:
Hi @qwordy , yes you need to add your subscription to an allowed list first. To make things easier, could you please email me the subscription ID we should add to make sure you are able to use our extension? That will make things easier.